.github/workflows/check-data-package-version.yml

```
@@ -0,0 +1,37 @@
+name: Check Data Package standard version
+
+on:
+  schedule:
+    # Runs at midnight on Sundays
+    - cron: 0 0 * * 0
+  workflow_dispatch:
+
+env:
+  CURRENT_VERSION: v2.0
+  DP_REPO: frictionlessdata/datapackage
+
+jobs:
+  check-latest-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch latest version
+        run: |
+          latest_version=$(curl -s https://api.github.com/repos/$DP_REPO/releases/latest | jq -r .tag_name)
+          echo "latest_version=$latest_version" >> $GITHUB_ENV
+          echo "Latest version is $latest_version"
+
+      - name: Create an issue if a new version is available
+        if: env.latest_version != env.CURRENT_VERSION
+        run: |
+          title="New \`$DP_REPO\` version available: $latest_version"
+          body="A new version of [\`$DP_REPO\`](https://github.com/$DP_REPO) is available: $latest_version.
+            Please check if updating Sprout's copy of the Data Package standard is necessary.
+
+            If so:
+            - Replace the old JSON schema of the Data Package standard with the new one downloaded from the website.
+            - Update the current version of \`$DP_REPO\` in the '$GH_WORKFLOW' GitHub workflow."
```

```suggestion

            - [ ] Replace the old JSON schema of the Data Package standard with the new one downloaded from the website.
            - [ ] Update the current version of \`$DP_REPO\` in the '$GH_WORKFLOW' GitHub workflow."
```

Just so they are checklists.